### ISSUE TITLE
You Got a Skull

### ISSUE THREAD

**Issue by MCTyler** [Rocologo/MobHunting#360]
It keeps sayin that but I do not get a Skull. Version 5.5.0. Spigot 1.12.2
![2018-03-03_20 04 40](https://user-images.githubusercontent.com/6390898/36941360-c6859a4c-1f1e-11e8-8a48-938d2a318f80.png)
I really don't want to get a skull for every kill anyway, I would like it to be rare.

**Comment by Rocologo** [comment]
Please show me your settings for the horse

**Comment by MCTyler** [comment]
I already rolled back to the previous Release , but it wasn't just the horse, it was every single Mob and I didn't touch the configuration so what ever stock config is. I also deleted my review.

**Comment by Rocologo** [comment]
Thank you, I will of cause solve the issue, but I need to see the config.yml if you dont have it anymore, the please let me have the current, the I will try to convert it on my test server.

**Comment by MCTyler** [comment]
Just pull it straight off your source, like I said, I didn't touch it.

**Comment by MCTyler** [comment]
If you like I will extract your Jar and upload the config but this seems silly.

**Comment by Rocologo** [comment]
No no, it is your config.yml I need,  I do of cause have the JAR :-) I have done the conversion 100 times on my test server, I you have probably configured MobHunting different from me, so I need to see what happens when the old config.yml is converted into the new format.

**Comment by MCTyler** [comment]
EDIT: Ok, very interesting....It does not do this on the test server, Just when I uploaded the Jar to my Live Server that had the previous config file set up already. So I guess I need to delete all my configuration let the plugin load then re-enter all my MySql data ect. Sorry to bother you. I would reccommend making transistions to updates a more user friedly process or add a disclaimer that the config file needs to be deleted.

**Comment by Rocologo** [comment]
I have spend about 40 hours making a conversion. I have got help from to other testers and I have asked for more testers in the forum, just to make it more user friendly. 

I have installed the plugin on my own prod server and done it 100 times on my test sever. Both with conversion and without (default config) everything has worked.

Could it be that you have run the old jar on your server using the new config. 

Notice that I have made a backup of your config.yml in the backup folder so you always can go vback.
